run-spartacus-ccv2-e2es.sh

```diff
@@ -1,5 +1,28 @@
 #!/bin/bash
 
+# uncomment below for local testing
+# export ENDPOINT_URL_PUBLIC_SPARTACUS=https://spartacusstore.cg79x9wuu9-eccommerc1-p1-public.model-t.myhybris.cloud
+# export ENDPOINT_URL_PUBLIC_API=https://api.cg79x9wuu9-eccommerc1-p1-public.model-t.myhybris.cloud
+
+CYPRESS_ROOT_FOLDER="cypress/integration"
+CYPRESS_B2C_FOLDER="$CYPRESS_ROOT_FOLDER/regression"
+CYPRESS_B2B_FOLDER="$CYPRESS_ROOT_FOLDER/b2b/regression"
+CYPRESS_ACCESSIBILITY_FOLDER="$CYPRESS_ROOT_FOLDER/accessibility"
+CYPRESS_SSR_FOLDER="$CYPRESS_ROOT_FOLDER/ssr"
+CYPRESS_VENDOR_FOLDER="$CYPRESS_ROOT_FOLDER/vendor"
+
```

Comment:
These 4 variables don't look like they are used.  Are they there on purpose?

Reply:
Yup, I left there on purpose for future usage.

Can remove if you want, but I was thinking it would ease any new implementor to just use the available variables

Reply:
Sounds good.  Let's leave them there is they have a purpose.